src/backend/columnar/mod.c

```diff
@@ -41,8 +37,6 @@ columnar_init(void)
 void
 columnar_fini(void)
 {
-	cstore_fdw_finish();
-
 #if HAS_TABLEAM
```

Comment:
should we restructure the includes? in pg11 we don't have to initialiase anything for columnar anymore right?